packages/manager/modules/pci/src/components/project/storages/databases/database.class.js

```diff
@@ -116,7 +116,10 @@ export default class Database extends Base {
 
   updateData(data) {
     Object.assign(this, data);
-    this.nodes = data.nodes.map((node) => new Node(node));
+    // only update nodes if they are provided in data
+    if (data.nodes) {
+      this.nodes = data.nodes.map((node) => new Node(node));
+    }
```

Comment:
```suggestion
    this.nodes = data.nodes?.map((node) => new Node(node)) || [];
```

Reply:
I didn't see that nodes was already set higher in this file, so this comment is not necessary